settings.gradle

```diff
@@ -110,6 +109,11 @@ dependencyResolutionManagement {
             }
         }
     }
+    versionCatalogs {
+        libs {
+            from("net.corda:corda-version-catalog:$cordaVersionCatalog")
```

Comment:
Since version catalog is going to have a bigger impact on the multiple repositories, possibly it would make sense to **announce** it is coming first to a wider Engineering group?

Especially given that it will affect the way we manage 3rd party dependencies going forward.